### DISCUSSION TITLE
k8s coordinator ip

### DISCUSSION THREAD

**Discussion by pilipyukaaa** [trinodb/trino#19380] [Q&A]
hello
into worker config i have value
discovery.uri=
in docs (https://trino.io/docs/current/security/internal-communication.html) you tels that i need only an ip for ssl connction
'discovery.uri=https://coordinator ip address:https port'
but, im deployed my trio cluster into k8s
please help me find mechanism for getiing coordinator ip in k8s

**Comment by luhea** [comment]
if you use helm deploy , this configuration did not configurate
